README.md

```diff
@@ -42,21 +42,21 @@ binding to that enum.
 Explore all of the tools this library comes with by checking out the [documentation][docs], and
 reading these articles:
 
-* **[What is navigation?][what-is-article]**:
```

Comment:
Can you update the short links at the bottom of the README instead?

Reply:
Done! That's a handy markdown feature with the short links!